src/components/accession2/viabilityTesting/TableCellRenderer.tsx

```diff
@@ -76,14 +76,22 @@ export default function Renderer(props: RendererProps<TableRowType>): JSX.Elemen
   }
 
   if (column.key === 'viabilityPercent') {
-    const { viabilityPercent } = row;
-    if (!row.endDate || row.viabilityPercent === undefined) {
+    const { testType, viabilityPercent, seedsFilled, seedsCompromised, seedsTested } = row;
+
+    const getViabilityPercent = () => {
+      if (testType !== 'Cut') {
+        return viabilityPercent;
+      }
+      return Math.floor(((seedsFilled + seedsCompromised) / seedsTested) * 100);
```

Comment:
Not related to this PR, but should we show this viability value in the viability modal suggestion that appears after saving a completed test? It's not appearing right now for cut tests because it validates that test is completed and cut test can't be marked completed

Reply:
I was thinking about this.. we need to ask PMs or Ming maybe. I'll post this on Figma.

Reply:
we can follow up separately